etc/tendrl/notifier_logging.yaml.sample

```diff
@@ -16,13 +16,11 @@ handlers:
         class: logging.handlers.TimedRotatingFileHandler
         level: INFO
         formatter: simple
-        filename: /var/log/tendrl/notifier/notifier_info.log
```

Comment:
ahh, this file is wrong, this is not syslog, i have to change this file as syslog conf :)